frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json

```diff
@@ -87,15 +87,15 @@
   "Time to live": "Time to live",
   "Time to live is the time that an object is stored in a caching system before it is deleted or refreshed. Default: 1 hr, Min: 15 mins, Max: 24 hrs": "Time to live is the time that an object is stored in a caching system before it is deleted or refreshed. Default: 1 hr, Min: 15 mins, Max: 24 hrs",
   "Read NamespaceStores": "Read NamespaceStores",
-  "Select list of namespace stores, defines the read targets of the namespace bucket": "Select list of namespace stores, defines the read targets of the namespace bucket",
+  "Select a list of NamespaceStore which defines the read targets of the namespaced bucket": "Select a list of NamespaceStore which defines the read targets of the namespaced bucket",
```

Comment:
Select a list of Namespace stores that defines the read targets of the namespace bucket.